indium-nodejs.el

```diff
@@ -113,6 +114,20 @@ If no process has been started, or if it was not started using
                 (read-from-minibuffer "Path: ")))
   (indium-nodejs--connect host port path))
 
+(defun indium-nodejs-connect-to-url (url)
```

Comment:
Could you make this function private and move it down in the file next to other private functions?

Reply:
Should this function be necessary if someone want to use directly (like from scratch buffer?)

(indium-nodejs-connect-to-url "...") [eval]